backend/app/queries/spree/admin/reports/average_order_values_query.rb

```diff
@@ -33,8 +33,15 @@ def group_by_date
         end
 
         def apply_date_filters(scope)
-          scope = scope.where('completed_at >= ?', opts[:completed_at_min]) if opts[:completed_at_min]
-          scope = scope.where('completed_at <= ?', opts[:completed_at_max]) if opts[:completed_at_max]
+          if opts[:completed_at_min]
```

Comment:
Why do we need these changes? They're out of the scope of `SR-24` task.

Reply:
because not having that change breaks the specs on CI (timezone issues)